README.md

```diff
@@ -64,6 +64,8 @@ Refer to the [building instructions](BUILDING.md) to compile SoH.
 - Affirm that you have an `/assets` folder filled with XMLs in the same directory as OTRGui.exe
 - Affirm that `zapd.exe` exists in the `/assets/extractor` folder
 
+## Nightly Builds
+ If You want the latest version of soh go to https://builds.shipofharkinian.com/
```

Comment:
Try something more along the lines of:

The latest builds are available at \[The Ship of Harkinian website\]\(https://builds.shipofharkinian.com/)
or:
"Nightly builds of Ship of Harkinian are available at"


Also, nobody is going to actually be able to find a downloadable build at that link. And I say that with some authority because, I can't figure out how to download the build at the link.

Reply:
There's probably a jenkins plugin you can get that lists all the artifacts at a single link.